### PR TITLE
add-paypal-button-to-widget-section

### PR DESCRIPTION
**Pull request by lukasz-matysiewicz**

### CODE REVIEW

File: blog/themes/3mdeb/layouts/partials/widget-donate.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="card">
```
Comment (artur-rs) [inline blog/themes/3mdeb/layouts/partials/widget-donate.html]: @lukasz-matysiewicz on my local preview the donate button moved awhole section with the rest plugins to the right

**Comment by artur-rs** [comment]
@lukasz-matysiewicz missing the plugin on the main page: `blog/themes/3mdeb/layouts/index.html`

**Comment by artur-rs** [comment]
@lukasz-matysiewicz we should cleanup some templates - there are some duplicated ones from `layouts` and `layouts/taxonomies`. Please test which ones are necessary and delete the redundant files.

**Comment by lukasz-matysiewicz** [comment]
> @lukasz-matysiewicz we should cleanup some templates - there are some duplicated ones from `layouts` and `layouts/taxonomies`. Please test which ones are necessary and delete the redundant files.

done

**Comment by lukasz-matysiewicz** [comment]
> lugin on the main page: `blog/th`

done

**Comment by lukasz-matysiewicz** [comment]
> @lukasz-matysiewicz missing the plugin on the main page: `blog/themes/3mdeb/layouts/index.html`

done

**Pull request merged**